frontend/src/components/VendorContacts.js

```diff
@@ -22,7 +22,7 @@ const VendorContacts = (props) => {
     {
       title: website,
       logo: webLogo,
-      link: 'https:'
+      link: '//'
```

Comment:
Nem kell prefix. Semennyire.

Reply:
Próbáltam úgy is, de localhoston így jelenne meg a link, ha kiveszek minden prefixet: http://localhost:3000/arusok/www.justincase.com

Reply:
Igen, mert a DB-dben www.justincase.com van. Én meg a teljes URL-t fogom beírni: https://www.justincase.com

Reply:
És akkor a //https://www.justincase.com már invalid lesz.

Reply:
Rendben.